### PR TITLE
mcproxy: some add some fixes

### PR DESCRIPTION
**Pull request by blogic**
This PR contains 2 patches provided by members of prpl.

### CODE REVIEW

**Comment by ecsv** [comment]
@stepet92 @Noltari @sbyx @mwarning

File: mcproxy/patches/0006-block-ingress.patch [hunk 1]
```diff
@@ -0,0 +1,104 @@
+--- a/mcproxy/src/proxy/proxy_instance.cpp
++++ b/mcproxy/src/proxy/proxy_instance.cpp
+@@ -171,6 +171,9 @@ void proxy_instance::worker_thread()
+     HC_LOG_TRACE("");
+     while (m_running) {
+         auto msg = m_job_queue.dequeue();
++
++		HC_LOG_DEBUG("Proxy Message: " << msg->get_message_type_name(msg->get_type()) );
```
Comment (mwarning) [inline mcproxy/patches/0006-block-ingress.patch]: Did a tab sneak in here?

**Comment by blogic** [comment]
fixed the mixed indentation and the  unrelated tab fix.

File: mcproxy/patches/0006-block-ingress.patch [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
++
+         switch (msg->get_type()) {
+         case proxy_msg::TEST_MSG:
+             (*msg)();
+@@ -193,25 +196,69 @@ void proxy_instance::worker_thread()
+         }
+         break;
+         case proxy_msg::GROUP_RECORD_MSG: {
+-            auto r =  std::static_pointer_cast<group_record_msg>(msg);
++            auto gr =  std::static_pointer_cast<group_record_msg>(msg);
```
Comment (mwarning) [inline mcproxy/patches/0006-block-ingress.patch]: There is a double white space after the equal sign. Better get rig of it.

File: mcproxy/patches/0006-block-ingress.patch [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+-                std::cout << *r << std::endl;
++                std::cout << *gr << std::endl;
+                 std::cout << std::endl;
+             }
+ 
+-            auto it = m_downstreams.find(r->get_if_index());
++            auto slist = gr->get_slist();
++            addr_storage saddr;
++            if ( slist.empty() )
++            {
```
Comment (mwarning) [inline mcproxy/patches/0006-block-ingress.patch]: The code style changes here. Better keep in uniform.

Comment (blogic) [reply mcproxy/patches/0006-block-ingress.patch]: the mcproxy code is littered with 25 different coding styles, but if it makes you happy :-)

**Comment by blogic** [comment]
added a 2nd patch to fix a compile warning in patch 0003

**Comment by mwarning** [comment]
It might sense to also direct the changes upstream.

**Comment by mwarning** [comment]
@sbyx please merge if you have no objections.

File: mcproxy/patches/0006-block-ingress.patch (hunk 3 above)
Comment (mwarning) [reply mcproxy/patches/0006-block-ingress.patch]: ok, it is one of those projects. :P

**Comment by mwarning** [comment]
@sbyx do you need more time? Otherwise I would like to merge it in about a week if you don't mind.

**Comment by mwarning** [comment]
A wild jow appears - thanks. :-)

**Pull request merged**